docs/data-guidelines.md

```diff
@@ -142,3 +142,10 @@ This decision was made in [#3953, under the expectation that most users are like
 For Safari for iOS, use the iOS version number, not the Safari version number or WebKit version number.
 
 This versioning scheme came at [Apple's request, in #2006](https://github.com/mdn/browser-compat-data/issues/2006#issuecomment-457277312).
+
+## Removal of irrelevant data
```

Comment:
A suggestion to retitle this. I think we have different standards for browser data, so I think it would be good to narrow the scope a little bit.

```suggestion
## Removal of irrelevant features
```